### PR TITLE
wip: mv density from struct. mat. to phys. mat.

### PR DESCRIPTION
**Pull request by jchick-bentley**
for the QTO effort, I moved the density property from the base aspect in the StructuralMaterials schema to the PhysicalMaterial in BisCore.

### CODE REVIEW

File: Domains/StructuralMaterials/StructuralMaterials.ecschema.xml [hunk 1]
```diff
@@ -11,7 +11,6 @@
     </ECEntityClass>	
     <ECEntityClass typeName="GenericMechPropertiesAspect" description="Mechanical Properties for a Generic Material (i.e. other than Aluminum, Concrete or Steel)" displayLabel="Generic Structural Mechanical Properties" modifier="None">
         <BaseClass>MechPropertiesAspect</BaseClass>
-        <ECProperty propertyName="Density" typeName="double" description="Density" displayLabel="Density" kindOfQuantity="AECU:DENSITY" />
```
Comment (laurynas111) [inline Domains/StructuralMaterials/StructuralMaterials.ecschema.xml:14]: We've accidentally released out product with this schema. Would it be possible to mark this property as deprecated instead? And also update schema version to 1.0.1?

Comment (jchick-bentley) [reply Domains/StructuralMaterials/StructuralMaterials.ecschema.xml:14]: @laurynas111 done - just pushed.  Both good suggestions.

File: Domains/Core/BisCore.ecschema.xml [hunk 2]
```diff
@@ -1868,6 +1868,7 @@
 
     <ECEntityClass typeName="PhysicalMaterial" modifier="Abstract" displayLabel="Physical Material" description="A bis:PhysicalMaterial defines the matter of which physical objects are made. bis:PhysicalMaterials are not used to define rendering characteristics (see bis:RenderMaterial).">
         <BaseClass>DefinitionElement</BaseClass>
+        <ECProperty propertyName="Density" typeName="double" description="Density" displayLabel="Density" minimumValue="0.0" kindOfQuantity="AECU:DENSITY" />
```
Comment (ColinKerr) [inline Domains/Core/BisCore.ecschema.xml:1871]: do not duplicate property name in displayLabel if the invariant name is the same as the display label.  Also do not set the description to the display label or invariant name.  Only set the description if it adds meaning.

Consider writing some docs in the remarks.md to add detailed description of Density

Comment (jchick-bentley) [reply Domains/Core/BisCore.ecschema.xml:1871]: @ColinKerr Thank you - taking care of both comments r.n.

File: Domains/Core/BisCore.remarks.md [hunk 3]
```diff
@@ -283,6 +283,12 @@ The system handler also restricts deletion. A `GeometryPart` can only be deleted
 
 > Behavior: The system handler restricts deletion. A `Texture` can only be deleted via a more expensive `deleteDefinitionElements` method that has determined the `Texture` is no longer referenced.
 
+### PhysicalMaterial
+
+The `PhysicalMaterial` serves as the base class for the materials in the PhysicalMaterial schema.  Generally, it is expected that domain specific properties e.g. strength properties for the structural domain will be added to the subclasses such as Concrete and Steel using domain specific ElementAspects.
```
Comment (caseymullen) [inline Domains/Core/BisCore.remarks.md:288]: domain-specific should be hyphenated. Parens around the "e.g" phrase.

File: Domains/Core/BisCore.remarks.md [hunk 4]
```diff
@@ -283,6 +283,12 @@ The system handler also restricts deletion. A `GeometryPart` can only be deleted
...
+
+The Density property in this base `PhysicalMaterial` class is intended to provide basic weight computation for the purposes of quantity takeoff and carbon footprint for example which are cross domain functions.
```
Comment (caseymullen) [inline Domains/Core/BisCore.remarks.md:290]: "cross-domain" hyphenated. Maybe parens around "for example"

**Review by caseymullen — CHANGES_REQUESTED** [review]

File: Domains/Core/BisCore.remarks.md (hunk 4 above)
Comment (jchick-bentley) [reply Domains/Core/BisCore.remarks.md]: > "cross-domain" hyphenated. Maybe parens around "for example"

put commas instead - thx

**Review by caseymullen — APPROVED** [review]

**Review by ColinKerr — DISMISSED** [review]

**Pull request merged**